src/main/java/com/edu/BaseballGame.java

```diff
@@ -0,0 +1,39 @@
+package com.edu;
+
+import java.util.Scanner;
+
+public class BaseballGame {
+
+    private static final String NUMBER_INPUT_MESSAGE = "숫자를 입력해주세요 : ";
+    private static final String GAME_ANSWER_MESSAGE = "3 스트라이크\n3개의 숫자를모두" +
+            "맞히셨습니다! 게임종료\n게임을새로시작하려면 1,종료하려면 2를입력하세요.";
+
+    public void playBaseballGame(Scanner scan) {
+        Hint hint = new Hint();
+        Restart restart = new Restart();
+        ComputerNumber computerNumber = new ComputerNumber();
+        String comNumber = computerNumber.createComNumber();
+        boolean isGameProgressCheck = true;
+
+        while (isGameProgressCheck) {
```

Comment:
```suggestion
        String comNumber = computerNumber.createComNumber();
        
        boolean isGameProgressCheck = true;
        while (isGameProgressCheck) {
```
while문이 반복되는 조건이 ``isGameProgressCheck`` 이므로 문맥상 둘은 붙여주는 것이 더 좋은 것 같습니다. 
또한 ``isGameProgressCheck`` 라는 네이밍보다는 조금 더 나은 네이밍이 있지 않을까요? (ex. **canPlay**) 

[참, 거짓을 반환 하는 함수의 네이밍](https://jacking75.github.io/ETC_true_false_return_function_naming/) 
[Boolean 변수명 짓는 팁](https://www.crocus.co.kr/1638)